torch/_torch_docs.py

```diff
@@ -1500,6 +1500,84 @@ def merge_dicts(*dicts):
     False
 """)
 
+add_docstr(torch.cov,
+           r"""
+cov(input, other=None, rowvar=True, bias=None, ddof=None, fweights=None, aweights=None) -> Tensor
+
+Estimates a covariance matrix, given input tensor data and weights (optional). The unbiased sample covariance 
+of the variables :math:`x` and :math:`y` is given by
```

Comment:
This is really cool and I like the mathematical expression (once the doc build completes we can even check the rendered LaTeX).

How do x and y relate to :attr:\`input\`, though?